### ISSUE TITLE
train problem

### ISSUE THREAD

**Issue by simuji** [dasayan05/sketchanet-quickdraw#2]
Thanks for the code you provided.
When I trained the sketch-a-net,I took 10 classes and batch-size of 8,but the loss count decrease all the time.How can I deal with it?
Thanks!

**Comment by Miao-kristoff** [comment]
Yes, what happen now? Have you delt with it?I find the same thing.

> Thanks for the code you provided. When I trained the sketch-a-net,I took 10 classes and batch-size of 8,but the loss count decrease all the time.How can I deal with it? Thanks!

**Comment by w-haiS** [comment]
> Yes, what happen now? Have you delt with it?I find the same thing.
> 
> > Thanks for the code you provided. When I trained the sketch-a-net,I took 10 classes and batch-size of 8,but the loss count decrease all the time.How can I deal with it? Thanks!

The same thing happened to me. Did you fix it？
